kclvm/tools/src/LSP/src/hover.rs

```diff
@@ -36,16 +36,28 @@ pub(crate) fn hover(
                         // attr2? type
                         // ```
                         let schema_ty = ty.into_schema_type();
-                        docs.push(schema_ty.schema_ty_signature_str());
+                        let schema_text = schema_ty.schema_ty_signature_str();
+
+                        for (i, ele) in schema_text.split("\n\n").enumerate() {
+                            if i == 0 {
+                                docs.push((ele.to_string(), "String"));
+                            } else {
+                                docs.push((ele.to_string(), "LanguageString"));
+                            }
+                        }
+
                         if !schema_ty.doc.is_empty() {
-                            docs.push(schema_ty.doc.clone());
+                            docs.push((schema_ty.doc.clone(), "String"));
                         }
 
+                        docs.push(("\n\n".to_string(), "String"));
+
                         // The attr of schema_ty does not contain the attrs from inherited base schema.
                         // Use the api provided by GlobalState to get all attrs
                         let module_info = gs.get_packages().get_module_info(&kcl_pos.filename);
                         let schema_attrs = obj.get_all_attributes(gs.get_symbols(), module_info);
-                        let mut attrs = vec!["Attributes:".to_string()];
+                        docs.push(("Attributes\n".to_string(), "String"));
```

Comment:
You dont need to follow the previous stype, and the `Attributes` is not must required
The previous hover content was:
\`\`\`
pkg
schema Name
\-\-\-
doc
\-\-\-
Attributes:
attr1: type1
attr2: type2
\`\`\`

Now can be rendered as

\`\`\`
pkg
\`\`\`kcl
schema Name:
    attr1: type1
    attr2: type2
\`\`\`
\-\-\-
doc

\`\`\`

Or some other better style.

Reply:
I have updated the formatting.